crates/bevy_input_focus/src/lib.rs

```diff
@@ -376,6 +385,9 @@ mod tests {
         };
         app.world_mut().spawn((window, PrimaryWindow));
 
+        // Run the world for a single frame to set up the initial focus
+        app.update();
```

Comment:
This was needed as we're now setting the input focus in a startup system, rather than inside the windowing event loop.